### PR TITLE
Chore/nav layout self hosted

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: studio/lib/constants/index.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ export * from './infrastructure'
 export * from './metrics'
 
 export const IS_PLATFORM = process.env.NEXT_PUBLIC_IS_PLATFORM === 'true'
+export const DEFAULT_HOME = IS_PLATFORM ? '/projects' : 'project/default'
```
Comment (alaister) [inline studio/lib/constants/index.ts:5]: Why does `/projects` have a leading `/`, but `project/default` does not?

File: studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx [hunk 2]
```diff
@@ -49,8 +49,8 @@ const NavigationBar = ({}) => {
       ].join(' ')}
     >
       <ul className="flex flex-col space-y-2">
-        {!navLayoutV2 && (
-          <Link href="/projects">
+        {(!navLayoutV2 || IS_PLATFORM) && (
```
Comment (alaister) [inline studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx:52]: Is this check correct? Getting the following on platform:

Screenshot 2023-07-31 at 3 16 00 pm

**Review by alaister — COMMENTED** [review]
Looking good! Couple of Qs:

File: studio/lib/constants/index.ts (hunk 1 above)
Comment (joshenlim) [reply studio/lib/constants/index.ts:5]: oh good catch! 🙏

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**